test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/UtilVerbs.java

```diff
@@ -508,7 +508,9 @@ public static HapiSpecOperation updateLargeFile(
 			OptionalLong tinyBarsToOffer
 	) {
 		return updateLargeFile(payer, fileName, byteString, signOnlyWithPayer, tinyBarsToOffer,
-				op -> {}, op -> {});
+				op -> {
+				}, op -> {
+				});
```

Comment:
This could be reverted to the previous implementation for consistency.

Reply:
Just my auto format, can revert to the original in another PR.